wcivf/apps/elections/tests/test_election_views.py

```diff
@@ -97,6 +97,25 @@ def test_division_name_displayed(self):
                     response, election.pluralized_division_name.title()
                 )
 
+    def test_election_type_filters(self):
+        """
+        Test that the election type filters work
+        when i click the 'Local' button, the queryset should only
+        contain referendum local elections
+        """
+        local_election = ElectionWithPostFactory(
+            slug="local.southfields.2022-06-23",
+            election_date="2022-06-23",
+            name="Southfields local election",
+        )
+        for election in [local_election]:
+            with self.subTest(election=election):
+                url = reverse("elections_view")
+                response = self.client.get(url, follow=True)
```

Comment:
I dont think that the test does what the docstring above describes, as the url used doesn't do any filtering - it is the main election index `/elections/`. So even though the `local_election` is returned as expected, you are not testing If you added in an assert to check that other election is not displayed, I would expect the test to fail.

If you want to use a URL that mimics having clicked a filter button, you will need to add in the relevant query string e.g:
```
url = reverse("elections_view")
url = f"{url}?{election.election_type}"
response = self.client.get(url, follow=True)
```
Which hopefully produces the url `/elections/?election_type=local`. 

Then to make the test more robust, you could create election objects for each election type, add them to your list on line 111, and assert that for each election type, object that election object is returned.